core/modules/qdisp/ExecStatus.h

```diff
@@ -57,7 +57,8 @@ class ExecStatus {
                  PROVISION_OK, // ???
                  REQUEST, REQUEST_ERROR,
                  RESPONSE_READY, RESPONSE_ERROR,
-                 RESPONSE_DATA, RESPONSE_DATA_ERROR, RESPONSE_DATA_NACK,
+                 RESPONSE_DATA, RESPONSE_DATA_ERROR, RESPONSE_DATA_ERROR_OK,
+				 RESPONSE_DATA_ERROR_CORRUPT, RESPONSE_DATA_NACK,
```

Comment:
I shifted things around. The formatting errors are caused by the editor using tabs instead of spaces, which I've fixed.